src/query.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import "github.com/newrelic/infra-integrations-sdk/data/metric"
+
+// Query defines a JMX query that has to be performed. Multiple JMX attributes can be received through a single Query.
+// Each JMX Attribute maps to a single NR metric. We set an Alias to attribute to define the name of the metric in NR.
+type Query struct {
```

Comment:
YAML fields will be required in a next PR will enable filtering by configuration